webpack/assets/javascripts/foreman_jed.js

```diff
@@ -0,0 +1,29 @@
+import Jed from 'jed';
+import { deprecate } from './foreman_tools';
+
+const locales = window.locales || {};
+let locale = document.getElementsByTagName('html')[0].lang;
+locale = locale.replace(/-/g, '_');
+const i18n = new Jed(locales[locale]);
+const cheveronPrefix = window.I18N_MARK ? '\u00BB' : '';
+const cheveronSuffix = window.I18N_MARK ? '\u00AB' : '';
+export const t_ = (...args) => `${cheveronPrefix} ${i18n.gettext(...args)} ${cheveronSuffix}`;
+
+window.__ = t_;
+window.n__ = (...args) => `${cheveronPrefix} ${i18n.ngettext(...args)} ${cheveronSuffix}`;
+
+Object.defineProperty(window, 'i18n', {
+  get: () => {
+    deprecate('i18n', 'tfm.i18n');
+    return i18n;
+  },
+});
+Object.defineProperty(window, 'Jed', {
```

Comment:
Does it make sense to add a new function to `foreman_tools` call `deprecateObjectProperty`?

```js
export const deprecateObjectProperty = (obj, oldMethod, newMethod, version = '1.19') =>
  Object.defineProperty(obj, oldMethod, {
    get: () => {
      deprecate(oldMethod, newMethod, version);
      return obj[oldMethod];
    },
  });

deprecateObjectProperty(window, 'i18n', 'tfm.i18n');
deprecateObjectProperty(window, 'Jed', 'tfm.i18n');
```